.gitignore

```diff
@@ -1,6 +1,5 @@
 node_modules/
 lib/
-dist/
```

Comment:
Since the files in the `dist` folder contain all dependencies baked into one file it's not really supposed to got into git.